src/pacer/api_integration.py

```
@@ -0,0 +1,398 @@
+#!/usr/bin/env python3
+"""
+PACER API integration for ch11.ai Flask server
+"""
+
+import asyncio
+import threading
+from datetime import datetime
```

Ruff (0.11.9)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)